.gitignore

```diff
@@ -345,3 +345,14 @@ ASALocalRun/
 common-*.tgz
 Chart.lock
 Remote.Payments.Internal.Gateway.Client/*.g.cs
+/Dojo.OpenApiGenerator.TestWebApi/OpenApiSchemas/20200103
```

Comment:
just out of curiosity, are you trying to load all json files? or there is some kind of convention? Or you are loading and checking if it is openapi compatible file?

Reply:
We will load all Json files that exist in the roots project "OpenApiSchemas" folder.
We also use .NET framework to convert to a valid OpenApiDocument, at which point it would fail with OpenApiUnsupportedSpecVersionException